src/pykeen/pipeline/api.py

```diff
@@ -915,6 +915,11 @@ def pipeline(  # noqa: C901
     if evaluator_kwargs is None:
         evaluator_kwargs = {}
     evaluator_kwargs = dict(evaluator_kwargs)
+    if evaluator_kwargs.get('filtered'):
+        all_pos_triples = torch.cat([training.mapped_triples, testing.mapped_triples], dim=0)
```

Comment:
From my point of view, we shouldn't use test triples for filtering during hyperparameter search.

Reply:
I agree 👍 
We should still address the case when training a single model and evaluate against the test set.

Reply:
I guess we can re-use the 
```python
use_testing_data: bool = True,
```
parameter to decide which triples should be used for filtering.

Reply:
Done in 3f0581b0a448270d6578fb9c43d68d4317424790.